test/sharness/t0110-gateway.sh

```diff
@@ -32,10 +32,6 @@ test_expect_success "GET IPFS path output looks good" '
   rm actual
 '
 
-test_expect_success "GET IPFS path on API forbidden" '
-  test_curl_resp_http_code "http://127.0.0.1:$apiport/ipfs/$HASH" "HTTP/1.1 403 Forbidden"
-'
-
```

Comment:
I thought this entire PR was reverted. I guess it was not because this test is not in master. 

cc @lgierth we need this back, i think.